CHANGELOG.md

```diff
@@ -41,7 +41,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Handle undefined attendeeId when calling `realtimeSetAttendeeIdPresence`
 - Fix `DefaultModality` base check
 - [Test] Fix a typo in integ tests
-- [Demo] Fix serverless deploy script to not print out logs
+- Fix getNearestMediaRegion link in README
```

Comment:
I guess you mistakenly removed the "[Demo] Fix serverless deploy script to not print out logs" fix from the changelog and also move "Fix getNearestMediaRegion link in README" to the unreleased section.